### DISCUSSION TITLE
winget cannot identify python3.

### DISCUSSION THREAD

**Discussion by teruteru128** [microsoft/winget-cli#1932] [Q&A]
I want to select `Python.Python.3` to upgrade, but I couldn't upgrade using such an option. What should i do?
```log
> winget upgrade --id Python.Python.3 -e
複数のインストール済みパッケージが一致する入力条件を検出しました。入力内容を修正してください。
名前                   ID
-------------------------------------------------------------
Python 3               Python.Python.3
Python 3.9.7 (64-bit)  {0f0bf1a5-3ec1-459b-ab7c-916db941f50d}
Python 3.10.1 (64-bit) {af822d5e-759c-4e77-9696-3cc835cd54a9}
```
winget version is `v1.1.13405`.

**Comment by OfficialEsco** [comment]
It does detect it as you can see by ID `Python.Python.3`, but you have 3 versions installed which causes a upgrade issue

**Reply by teruteru128** [reply]
Well, I uninstalled all Python3 from my system and installed Python3 again via winget.
